build/ci/deploy_driver.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xel
```

Comment:
can we not just dump all files to /build/ci? let's try to organize a little.. (relevant for all new files in that path.)

Reply:
do you have a good idea for a dedicated directory?
something like `build/ci/github_actions`?

Reply:
shouldn't it be combined with the [community deployment](https://github.com/IBM/ibm-block-csi-operator/pull/157#discussion_r651825784) tests (both in scenario and project dir)?

Reply:
the directory name `community` is not a related to what is done here, so that why I asked about `build/ci/github_actions` name

Reply:
shouldn't the driver build be done on the driver repo, and be found according to the branch name?

Reply:
I want that the naming will be identical as possible between the operator scripts and the driver scripts

Reply:
that how it works, I don't build the driver on the operator repository

Reply:
then maybe `build/ci/driver/` for the non `build/ci/community/` files?

Reply:
or even better: `build/ci/github_actions/community/` and `build/ci/github_actions/driver/`

Reply:
done

Reply:
@ArbelNathan do you approve?

Reply:
looks good, tnx